packages/hash/external-services/docker-compose.yml

```diff
@@ -44,19 +48,46 @@ services:
     ports:
       - "${POSTGRES_PORT}:5432"
     volumes:
-      - ../../../var/external-services/postgres/data:/data
+      - hash-postgres-data:/var/lib/postgresql/data
```

Comment:
I think this volume mount was done on purpose in external services, but I'm not sure what for. Just flagging we should check if there was a good reason for this to be mounted beforehand

Reply:
@benwerner01 do you know more about this?

Reply:
I'm not sure either

Reply:
@kachkaev do you know why this is mounting a directory rather than using a volume? The current way it's escalating root privileges and writes to `./var/external-services/...` with root rights.

Reply:
We discussed this with @TimDiekmann in Slack and agreed that we could switch to a volume. A local folder is used to make sure that all temp files are contained in the `var` folder. We upload this folder as a CI artifact for debugging. A volume will change this, but we can think of a fix later if we do need to inspect the context of the volume.